ft_artifact_threshold.m

```diff
@@ -161,7 +161,9 @@
   else
     dat = ft_read_data(cfg.datafile, 'header', hdr, 'begsample', cfg.trl(trlop,1), 'endsample', cfg.trl(trlop,2), 'chanindx', channelindx, 'checkboundary', strcmp(cfg.continuous, 'no'), 'dataformat', cfg.dataformat);
   end
-  dat = preproc(dat, channel, offset2time(cfg.trl(trlop,3), hdr.Fs, size(dat,2)), artfctdef);
+  if strcmp(artfctdef.bpfilter, 'yes')
```

Comment:
how do you know that only artfctdef.bpfilter might be set yes/no?  It might also be that some other filter is being used (e.g. lowpass).

Reply:
it should be fine If you would check that artfctdef is otherwise or that all fields are 'no'

Reply:
Since the reference document of ft_artifact_threshold only includes these specifications

% The following configuration options can be specified
%   cfg.artfctdef.threshold.channel   = cell-array with channel labels
%   cfg.artfctdef.threshold.bpfilter  = 'no' or 'yes'
%   cfg.artfctdef.threshold.bpfreq    = [0.3 30]
%   cfg.artfctdef.threshold.bpfiltord = 4

and the code from line 112 to 115 only considers the bpfilter, I reasoned that the application of a bandpass filter is the only selectable option. But of course I can expand the request.